server.js

```diff
@@ -8,10 +8,6 @@ const SERVICE_AGENT_ADMIN_BASE_URL =
   process.env.SERVICE_AGENT_ADMIN_BASE_URL ||
```

Comment:
Both `SERVICE_AGENT_ADMIN_BASE_URL` and `CREDENTIAL_DEFINITION_ID` are OK, but I don't think they should have a default value. I prefer users to explicitly define what is their Service Agent and the Credential Definition they want to request.